test/index.js

```diff
@@ -49,7 +48,7 @@ const testCommand = (command, expected) => {
     try {
       return {
         status: 0,
-        stdout: execSync (`${command} --nodejs --no-warnings`, options),
+        stdout: execSync (command, options),
```

Comment:
We no longer need to suppress warnings as ES modules are no longer experimental. :)